### PR TITLE
WinFeat version 1.5.1 by Fireseed

### PR DESCRIPTION
**Pull request by seedy60**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21168)

### CODE REVIEW

File: manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: FSN.WinFeat
+PackageVersion: 1.5.1
+PackageLocale: en-GB
+Publisher: Fireseed Network
```
Comment (ImJoakim) [inline manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.locale.en-GB.yaml:7]: The `Publisher` needs to match the one in the control panel:
image

```suggestion
Publisher: Fireseed
```

**Comment by fireseed66** [comment]
Why is the installer validation failing?

File: manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: FSN.WinFeat
+PackageVersion: 1.5.1
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.installer.yaml:7]:
```suggestion
- Architecture: x64
```

File: manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  InstallerType: inno
+  InstallerUrl: https://www.houseoffireseed.ml/programs/WinFeat/WinFeat_Installer.exe
+  InstallerSha256: 60CBBD2336480B2196C26D9A7D702310EC9E26F59B97EA5E8F16613C6FC29A97
```
Comment (ItzLevvie) [inline manifests/f/FSN/WinFeat/1.5.1/FSN.WinFeat.installer.yaml:10]:
```suggestion
  InstallerSha256: 60CBBD2336480B2196C26D9A7D702310EC9E26F59B97EA5E8F16613C6FC29A97
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**